tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1736,18 +1791,14 @@ def test_build_fails_with_missing_metadata(self, runtime, use_container, layer_i
         self.assertEqual(command_result.process.returncode, 1)
         self.assertFalse(self.default_build_dir.joinpath(layer_identifier).exists())
 
-    @parameterized.expand([("python3.7", False, "LayerOne"), ("python3.7", "use_container", "LayerOne")])
-    def test_build_with_missing_buildarchitecture(self, runtime, use_container, layer_identifier):
-        if use_container and (SKIP_DOCKER_TESTS or SKIP_DOCKER_BUILD):
-            self.skipTest(SKIP_DOCKER_MESSAGE)
-
-        overrides = {"LayerBuildMethod": runtime, "LayerContentUri": "PyLayer"}
+    def test_function_build_succeeds_with_referenced_layer(self):
+        overrides = {"Runtime": "python3.8"}
```

Comment:
Why did we replace the parameterized test (which test with and without container build) with constant one (which only test without container)?

Reply:
Ah, I didn't leave the parameterization out on purpose, I just forgot to re-add it after writing the new test. Will fix.